build/bin/release-next.sh

```diff
@@ -0,0 +1,59 @@
+#!/usr/bin/env bash
+
+echo "Release Type?"
+read -rp "> " rtype
+
+echo grunt version:${rtype}
+grunt version:${rtype}
+version=$(./build/bin/version)
```

Comment:
I'm not sure we want to allow that, though. I think some of the grunt scripts depend on being in the root.